src/witan/send/schemas.clj

```diff
@@ -42,10 +42,11 @@
                            [:population s/Int]]))
 
 (def SENDSchemaIndividual
-  (make-ordered-ds-schema [[:id s/Int]
-                           [:year (s/constrained s/Int u/year?)]
+  (make-ordered-ds-schema [[:year (s/constrained s/Int u/year?)]
                            [:age (s/constrained s/Int u/SENDage?)]
-                           [:state s/Keyword]]))
+                           [:state s/Keyword]
+                           [:sim-num s/Int]
+                           [:id s/Int]]))
```

Comment:
@seb231 because we're addind `:sim-num` and then `:id` I think they end up being the two last columns.